mono/mini/interp/transform.c

```diff
@@ -2126,7 +2295,7 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 		}
 	}
 
-	if (td->gen_sdb_seq_points) {
+	if (td->gen_sdb_seq_points && td->method == method) {
```

Comment:
does that mean we don't emit sequence points for inlined methods?

Reply:
I noticed that jit also doesn't do it so I didn't bother with it.

Reply:
with this commit (https://github.com/mono/mono/pull/11735/commits/464cb0146c76b43ad22ccedd24d855370cc3fa56) it's fine 👍